src/main/java/frc/robot/subsystems/drive/DriveSubsystem.java

```diff
@@ -142,20 +144,21 @@ public DriveSubsystem(SwerveModule frontLeft, SwerveModule frontRight, SwerveMod
                                                 frontRight.getPosition(),
                                                 rearLeft.getPosition(),
                                                 rearRight.getPosition() },
-                                new Pose2d(0, 0, new Rotation2d(0, 0))); // TODO: make these constants in the constants
-                                                                         // file rather than
-                                                                         // free-floating numbers
+                                new Pose2d(0, 0, new Rotation2d(0))); // TODO: make these constants in the constants
+                                                                      // file rather than
+                                                                      // free-floating numbers
 
-                configurePathPlannerLogging();
         }
 
         @Override
         public void periodic() {
                 // This will get the simulated sensor readings that we set
                 // in the previous article while in simulation, but will use
                 // real values on the robot itself.
-                SmartDashboard.putNumber("left front distance (meters)", frontLeft.getDriveEncoderPosition());
-                SmartDashboard.putNumber("drive/gyro angle(degrees)", Math.toDegrees(gyro.getYaw()));
+
+                SmartDashboard.putNumber("relative X speeds", relativeRobotSpeeds.vxMetersPerSecond);
```

Comment:
in the string label for these 3 items, prepend them with "drive/" so they get grouped with all the other logged items from this subsystem.